From 68ec4f69efd6fe366fadb23f61c8f153899bc3f7 Mon Sep 17 00:00:00 2001
From: Gerard Salvatella <gerard.salvatella@toradex.com>
Date: Tue, 27 Nov 2018 15:54:01 +0100
Subject: [PATCH] Makefile: change host compiler to gcc

Currently HOSTCC is set to cc. Change it to gcc in order for tezi to build.

Signed-off-by: Gerard Salvatella <gerard.salvatella@toradex.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 68614961f9..1601364de1 100644
--- a/Makefile
+++ b/Makefile
@@ -254,7 +254,7 @@ CONFIG_SHELL := $(shell if [ -x "$$BASH" ]; then echo $$BASH; \
 	  else if [ -x /bin/bash ]; then echo /bin/bash; \
 	  else echo sh; fi ; fi)
 
-HOSTCC       = cc
+HOSTCC       = gcc
 HOSTCXX      = c++
 HOSTCFLAGS   = -Wall -Wstrict-prototypes -O2 -fomit-frame-pointer \
 		$(if $(CONFIG_TOOLS_DEBUG),-g)
-- 
2.13.6

